Source/WebCore/platform/GStreamer.cmake

```diff
@@ -143,7 +141,6 @@ if (ENABLE_ENCRYPTED_MEDIA)
         ${LIBGCRYPT_INCLUDE_DIRS}
     )
     list(APPEND WebCore_SOURCES
-        platform/encryptedmedia/clearkey/CDMClearKey.cpp
```

Comment:
Why do you disable ClearKey in the OpenCDM case?
It can be kept, and is still useful to run generic EME tests.

Reply:
ClearKey is *NOT* disabled if you select OpenCDM. It is just running through the OpenCDM, to be handled in the OpenCDM server. If you deselect OpenCDM, the ClearKey is selected as the dafult DRM system, if you select OpenCDM, it is assumed that all DRM systems (including ClearKey) are running in the OpenCDM server)

Reply:
I don't see the ClearKey keysystem in WebKitOpenCDMDecryptorGStreamer.cpp caps.

Reply:
You just missed the last push :-)